workflow/json-visualizer/script.js

```diff
@@ -0,0 +1,91 @@
+const textarea = document.getElementById('text-area');
+const result = document.getElementById('result');
+const button = document.getElementById('button');
+const clearButton = document.getElementById('clear-button');
+
+button.addEventListener('click', handleButtonClick);
+clearButton.addEventListener('click', clearTextarea);
+
+function clearTextarea() {
+    textarea.value = '';
+    result.textContent = '';
+    result.innerHTML;
+}
+
+function parseJSONString(str) {
+    try {
+        return JSON.parse(str);
+    } catch (err) {
+        result.textContent = `Wrong format`;
+        result.innerHTML;
+        return null;
```

Comment:
we rewrited `renderResult ` to make all things connected to insert to `result.innerHTML`. Why don't you use it here?